src/components/child.js

```diff
@@ -21,10 +21,11 @@ class Child extends React.Component {
 
     console.log(this.props);
     return (
-      <strong>
-        {this.props.name} is {adjective}
-        and has {this.state.goldStars} gold stars!
-      </strong>
+      <>
+        <strong>
+          {this.props.name} is {adjective} and has {this.state.goldStars} gold stars!
+        </strong>
+      </>
```

Comment:
Using `<>` or `<React.Fragment>` instead of `<div>` or another real HTML element allows us to make JSX happy (return single element) without adding an extra real `<div>` to the DOM.